mne/beamformer/tests/test_dics.py

```diff
@@ -44,7 +44,7 @@ def _get_data(tmin=-0.11, tmax=0.15, read_all_forward=True, compute_csds=True):
     if read_all_forward:
         forward_surf_ori = _read_forward_solution_meg(fname_fwd, surf_ori=True)
         forward_fixed = _read_forward_solution_meg(fname_fwd, force_fixed=True,
-                                                   surf_ori=True)
+                                                   use_cps=False)
```

Comment:
is this necessary for the tests to pass?

FWIW these tests have never been all that robust. @britta-wstnr when you have time can you have a look at these to see if there is some way to make them more reliable without increasing (or while decreasing) test time?